python/chainer_compiler.py

```diff
@@ -162,6 +167,48 @@ def backward(self, indexes, flat_gys):
         return gxs
 
 
+class use_unified_memory:
```

Comment:
Could you use `contextlib.contextmanager`? https://docs.python.org/2/library/contextlib.html

Reply:
fixed. (`yield None` is OK in this case?)